// Copyright 2015 gRPC authors.
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//     http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.

syntax = "proto3";

option java_multiple_files = true;
option java_package = "com.michaelbutler.grpc.payment";
option java_outer_classname = "paymentServiceImpl";
option objc_class_prefix = "HLW";

package com.michaelbutler.grpc.payment;

// The greeting service definition.
service Greeter {
  // Sends a greeting
  	rpc paymentAmount (paymentAmountRequest) returns (paymentAmountReply) {}
  	rpc paymentDate (paymentDateRequest) returns (paymentDateReply) {}
}

// The request message containing the user's name.
message paymentAmountRequest {
  	string amount = 1;
}

// The response message containing the greetings
message paymentAmountReply {
 	string rAmount = 1;
}

// The request message containing the user's name.
message paymentDateRequest {
  	string date = 1;
}

// The response message containing the greetings
message paymentDateReply {
 	string rDate = 1;
}